pandas/core/groupby/groupby.py

```diff
@@ -1222,14 +1222,15 @@ def mean(self, *args, **kwargs):
         2    4.0
         Name: B, dtype: float64
         """
-        nv.validate_groupby_func("mean", args, kwargs, ["numeric_only"])
         return self._cython_agg_general(
-            "mean", alt=lambda x, axis: Series(x).mean(**kwargs), **kwargs
+            "mean",
+            alt=lambda x, axis: Series(x).mean(numeric_only=numeric_only),
+            numeric_only=numeric_only,
         )
 
     @Substitution(name="groupby")
     @Appender(_common_see_also)
-    def median(self, **kwargs):
+    def median(self, numeric_only=True, min_count=-1):
```

Comment:
can you update doc-strings for these parameters (or are they already ok with the Appenders?)